README.md

```diff
@@ -185,10 +185,13 @@ Warning! curl commands is subject for change
 
 * 'node' - node name as defined in pelagos configuration
 
+* 'add_sls' - add node specific salt script to provisioning. Mostly it is
```

Comment:
(suggestion) Why not ĵust simply `sls` instead of `add_sls`?

Reply:
replaced to extra_sls